numpydoc/__main__.py

```diff
@@ -1,5 +1,5 @@
 """
-Implementing `python -m numpydoc` functionality.
+Implementing `python -m numpydoc` functionality
```

Comment:
I think all docstrings (including module-level?) typically end with a period, no? In that sense this is a tiny regression. If it's for testing purposes (looks like it?) then a little comment here saying that's the case would help the next person not (mistakenly) fix it.

Reply:
Yeah, I can add a comment. Also happy to change to a different "defect" if we want.